app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessToiletsPart.kt

```diff
@@ -11,6 +11,7 @@ class AddWheelchairAccessToiletsPart : OsmFilterQuestType<WheelchairAccessToilet
 
     override val elementFilter = """
         nodes, ways with
+         wheelchair = limited
```

Comment:
have you tested it? From looking without testing I suspected that `and` is missing

Reply:
Thanks! I indeed did test it, but not this exact change...
This happens when you lazily type changes into github from memory